docs/_manage/account-and-team/add-team.md

```diff
@@ -8,33 +8,63 @@ redirect_from:
     - /manage/invite-team-member
 ---
 
-# Add team members to your integration
+# Invite Team Members to Your Integration
 
-You can add team members to your integration to collaborate, contribute, and get analytics data on the integration in the Zapier Platform. You can have up to 100 team members on an integration regardless of if it is a private or a public integration. Team members added to your Zapier integration can be assigned either of the following two roles:
+You can add team members to your integration to collaborate, contribute, and view analytics data for your integration on the Developer Platform. Your integration team can have up to 100 team members, regardless of whether your integration is Private or Public. Team members added to your integration will be assigned one of the following roles:
 
-- **Admin**: Admins have full access to build, manage, and edit your Zapier integration. Admin access is recommended for developers and others who directly contribute to building and maintaining your integration.
+- **Admin**: Admins are granted read and write access to the integration. This role is ideal for developers and individuals actively involved in building and maintaining the integration.
 
-- **Collaborator**: Collaborators have view-only access to your Zapier integration (including integration details, history, and analytics). Collaborators are unable to edit but can view performance data, view the integration history, comment on feature requests and bugs, and access tools to embed Zapier integrations inside your UI to rack up user adoption. Collaborator access is recommended for product, leadership, partnership, marketing, and other teams that do not contribute directly to the creation and maintenance of the integration, but need to get data regarding the integration for analysis and record purposes.
+- **Collaborator**: Collaborators are granted read-only access to the integration. Collaborator access is recommended for product, leadership, partnership, marketing, and other teams seeking access to integration data for analysis and record-keeping without direct involvement in its creation and maintenance. While they cannot make direct changes to the integration, they can:
 
-## Adding team members
```

Comment:
Add team members

Reply:
This was taken out, nothing to update here